Added previously proposed line on ssl in #3089 (#4592)
EnterpriseDB/docs: product_docs/docs/pgd/4/harp/04_configuration.mdx

- **`ssl_key_file`**: Client SSL key file.

> Note that when `ssl` is set to `on`, the etcd endpoint URLs must contain the `https` scheme, e.g. `https://host1:2379`.

Reviewer: ```suggestion
!!! Tip
    When `ssl` is set to `on`, the etcd endpoint URLs must contain the `https` scheme. For example, `https://host1:2379`.
```
